docs/optimization-workbook/README.md

```diff
@@ -10,4 +10,19 @@
 
 The cost optimization workbook is an Azure Monitor workbook that provides a single pane of glass for cost optimization, modeled after the Well-Architected Framework guidance.
 
+
+## Workbook overview
 We'll share more details as the workbook evolves.
+
+
+
+### Deployment
+* Consider the following least-privilege (minimum) RBAC roles when deploying the workbook:
+
+  * **Reader** and **Workbook Contributor** : allows you to import the workbook, view all of the workbook tabs and save the workbook in a resource group.
+  * **Reader** : allows you to import the workbook and view all of the workbook tabs.
+
+Use the following link to deploy the Optimization Workbook:
+
+   <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Farthurclares%2FWAFOptimizationFactory%2Fmain%2FOptmizationFactoryWorkbook%2Fazuredeploy.json" target="_blank"><img src="https://aka.ms/deploytoazurebutton"/></a>
```

Comment:
I don't want to link out to an external repo given it'll be different than what we have here. Let's remove this for now until we get it deployed to the Azure Quickstart Templates repo. I can do that as soon as this PR is closed.